test/plugins/alias-completion.plugin.bats

```diff
@@ -23,9 +30,13 @@ load ../../completion/available/capistrano.completion
   assert_success
 }
 
-@test "alias-completion: See that having aliased rm command does not output unnecessary output" {
-  alias rm='rm -v'
+@test "alias-completion: See that hidden behind function rm command does not change behavior" {
+  rm() {
+    touch test.file;
+    command rm $1;
```

Comment:
This might need to be `command rm "$@"` for completeness

Reply:
Oh I didn't realize how local this function is to the test ... `$1` is probably fine

Reply:
I will change this, so it wont break on future unknown tests for some reason, nice catch!